hawkauthlib/noncecache.py

```diff
@@ -73,7 +70,7 @@ def set(self, key, value, timestamp=None):
                 # Purge a few expired items to make room.
                 # Don't purge *all* of them, so we don't pause for too long.
                 for _ in range(5):
-                    (old_timestamp, old_key) = self.purge_queue[0]
+                    (old_timestamp, _old_key) = self.purge_queue[0]
```

Comment:
Is the leading `_` just because the value is not used?  If so, I wouldn't mind seeing it as just `(old_timestamp, _) = ...` to make that even clearer, but I don't feel strongly about it.

Reply:
yes, its about 'not used' .. pylint spits out warnings if you have unused values .. you can mask such unused value by '_foo' or even '_'. In other projects, I have seen that some guys use '__' (e.g. https://github.com/sphinx-doc/sphinx/blob/master/sphinx/application.py#L37) that's why I sometime hesitate  to use simple underline. But if you prefer a simple underline, I can fix it that way.

Reply:
👍  I don't feel strongly about this so I'm happy to leave it as-is.